src/zh/zazhimi/src/eu/kanade/tachiyomi/extension/zh/zazhimi/Zazhimi.kt

```diff
@@ -0,0 +1,105 @@
+package eu.kanade.tachiyomi.extension.zh.zazhimi
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import keiyoushi.utils.parseAs
+import okhttp3.HttpUrl.Companion.toHttpUrl
+import okhttp3.Request
+import okhttp3.Response
+
+class Zazhimi : HttpSource() {
+
+    private val apiUrl = "https://android2024.zazhimi.net/api"
+
+    override val baseUrl = "https://www.zazhimi.net"
+    override val lang = "zh"
+    override val name = "杂志迷"
+    override val supportsLatest = false
+
+    override fun headersBuilder() = super.headersBuilder()
+        .set("User-Agent", "ZaZhiMi_5.3.0")
+
+    // Popular
+
+    override fun popularMangaRequest(page: Int) = GET("$apiUrl/index.php?p=$page&s=20", headers)
+
+    override fun popularMangaParse(response: Response): MangasPage {
+        val result = response.parseAs<IndexResponse>()
+        val mangas = result.new.map(NewItem::toSManga)
+        return MangasPage(mangas, mangas.isNotEmpty())
+    }
+
+    // Latest
+
+    override fun latestUpdatesRequest(page: Int) = throw UnsupportedOperationException()
+
+    override fun latestUpdatesParse(response: Response) = throw UnsupportedOperationException()
+
+    // Search
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val url = apiUrl.toHttpUrl().newBuilder()
+            .addPathSegment("search.php")
+            .addQueryParameter("k", query)
+            .addQueryParameter("p", page.toString())
+            .addQueryParameter("s", "20")
+        return GET(url.build(), headers)
+    }
+
+    override fun searchMangaParse(response: Response): MangasPage {
+        val result = response.parseAs<SearchResponse>()
+        return MangasPage(result.magazine.map(SearchItem::toSManga), true)
+    }
+
+    // Manga Detail Page
+
+    override fun mangaDetailsRequest(manga: SManga) = GET(apiUrl + manga.url, headers)
+
+    override fun mangaDetailsParse(response: Response): SManga {
+        val result = response.parseAs<ShowResponse>()
+        if (result.content.isEmpty()) return SManga.create()
+        val item = result.content[0]
+        return SManga.create().apply {
+            title = item.magName
+            author = item.magName.split(" ")[0]
+            thumbnail_url = item.magPic
+            url = "/show.php?a=${item.magId}"
```

Comment:
```suggestion
            url = "/show.php?a=${item.magId}"
            update_strategy = UpdateStrategy.ONLY_FETCH_ONCE
```
since there is always one chapter to return, (meaning chapter list doesn't change), set update strategy so it is excluded from library update